iphone/Classes/TiViewProxy.m

```diff
@@ -2330,15 +2330,7 @@ -(void)relayout
         }
 
         if (layoutChanged && [self _hasListeners:@"postlayout" checkParent:NO]) {
-            
```

Comment:
We should probably leave this in, as it was for the issue when calling the postLayout we couldn't get the properties on on main thread